.github/CONTRIBUTING.md

```diff
@@ -58,35 +58,43 @@ These guides will walk your through various activities for contributing:
 
 ## Accessibility
 
+Stardust implements accessibility using accessibility behaviors. The behaviors add attributes to the DOM elements (mainly role and aria-* properties) as well as handle keyboard interaction and focus. Every accessible component has a defiault behavior, which can be overriden using the `accessibility` prop.
+
+Behaviors apply properties, focus handling and keyboard handlers to the component slots. When developing a component, the properties and keyboard handlers need to be spreaded to the corresponding slots.
+
+For complete accessibility contributing guide, requireements and testing, see [Accessibility][2]
+
 - [Role and aria props](#role-and-aria-props)
 - [Focus](#focus)
+- [Keyboard handling](#keyboard-handling)
 
 ### Role and aria props
 
-ARIA landmarks are attributes you can add to elements in your page to define areas like the main content or a navigation region.
-It is used to help the screen readers to find the correct elements on the page, where the semantic HTML is not able to cover
-it (like for elements of type pop-up).
-
-It is represented through the `role` and `aria-*` attributes which are usually added to divs and spans.
+ARIA [roles](https://www.w3.org/TR/wai-aria-1.1/#usage_intro) and [attributes](https://www.w3.org/TR/wai-aria-1.1/#introstates) provide information for screen readers to allow the users to navigate in the page/application.
```

Comment:
seems that we may be more general here, as this screen readers doesn't cover the full set of technologies that consume ARIA attributes/roles. Something like

> provide necessary semantics for assistive technologies that allow persons with disabilities to to navigate in the page/application.

Reply:
please add references to all the urls at the end of the file